api/src/main/java/com/codeforcommunity/rest/subrouter/TeamsRouter.java

```diff
@@ -22,12 +28,137 @@ public TeamsRouter(ITeamsProcessor processor) {
   @Override
   public Router initializeRouter(Vertx vertx) {
     Router router = Router.router(vertx);
-
+    registerCreateTeam(router);
+    registerGetTeam(router);
+    registerAddGoal(router);
+    registerDeleteGoal(router);
     registerDisbandTeam(router);
+    registerApplyToTeam(router);
+    registerInviteUser(router);
+    registerApproveUser(router);
+    registerRejectUserRoute(router);
+    registerGetApplicants(router);
+    registerKickUserRoute(router);
+    registerLeaveTeamRoute(router);
+    registerTransferTeamOwnership(router);
 
     return router;
   }
 
+  // Create a Team
+  private void registerCreateTeam(Router router) {
+    Route createTeamRoute = router.post("/create");
+    createTeamRoute.handler(this::handleCreateTeamRoute);
+  }
+
+  private void handleCreateTeamRoute(RoutingContext routingContext) {
+    JWTData userData = routingContext.get("jwt_data");
+    CreateTeamRequest createTeamRequest =
+        RestFunctions.getJsonBodyAsClass(routingContext, CreateTeamRequest.class);
+    processor.createTeam(userData, createTeamRequest);
+    end(routingContext.response(), 200);
+  }
+
+  // Get a Team
+  private void registerGetTeam(Router router) {
+    Route getTeamRoute = router.get("/:team_id");
+    getTeamRoute.handler(this::handleGetTeamRoute);
+  }
+
+  private void handleGetTeamRoute(RoutingContext routingContext) {
```

Comment:
Our convention is to call the RoutingContext ctx. Find and replace routingContext with ctx would fix that easily